app/templates/src/static/css/brand-palette.less

```diff
@@ -3,28 +3,67 @@
 
-@green: #2CB34A;
-@green-midtone: #ADDC91;
-@green-tint: #DBEDD4;
+// Primary colors:
 
-@white: #FFFFFF;
-@black: #101820;
-@darkgray: #43484E;
+@green:             #2CB34A;
+@green-midtone:     #ADDC91;
+@green-tint:        #DBEDD4;
 
-// Base                 // 80% Tint                // 50% Tint                // 20% Tint
-@redorange: #D12124;    @redorange-80: #DA6750;    @redorange-50: #E8A091;    @redorange-20: #F6D9D3;
-@gold: #FF9E1B;         @gold-80: #FFB149;         @gold-50: #FFCE8D;         @gold-20: #FFECD1;
-@neutral: #796E65;      @neutral-80: #948B84;      @neutral-50: #BCB6B2;      @neutral-20: #E4E2E0;
-@navy: #002D72;         @navy-80: #33578E;         @navy-50: #7F96B8;         @navy-20: #CCD5E3;
-@pacific: #0072CE;      @pacific-80: #328ED8;      @pacific-50: #7FB8E6;      @pacific-20: #CCE3F5;
-@teal: #005E5D;         @teal-80: #337E7D;         @teal-50: #7FAEAE;         @teal-20: #CCDFDF;
+@black:             #101820;
 
-@gray: #75787B;         @gray-80: #919395;         @gray-50: #BABBBD;         @gray-20: #E3E4E5;
-                                                                              @gray-10: #F1F2F2;
-                                                                              @gray-5: #F8F8F8;
 
-// Unofficial or proposed colors
+// Background colors:
 
-@dark-redorange: #9C301B; // Needed for active state of destructive action buttons
+@darkgray:          #43484E; // Base
+
+@gray:              #75787B; // Base
+@gray-80:           #919395; // 80% tint
+@gray-50:           #BABBBD; // 50% tint
+@gray-20:           #E3E4E5; // 20% tint
+@gray-10:           #F1F2F2; // 10% tint
+@gray-5:            #F8F8F8; // 05% tint
+
+
+// Secondary colors:
+
+@redorange:         #D12124; // Base
+@redorange-80:      #DA6750; // 80% tint
+@redorange-50:      #E8A091; // 50% tint
+@redorange-20:      #F6D9D3; // 20% tint
+
+@gold:              #FF9E1B; // Base
+@gold-80:           #FFB149; // 80% tint
+@gold-50:           #FFCE8D; // 50% tint
+@gold-20:           #FFECD1; // 20% tint
+
+@neutral:           #796E65; // Base
+@neutral-80:        #948B84; // 80% tint
+@neutral-50:        #BCB6B2; // 50% tint
+@neutral-20:        #E4E2E0; // 20% tint
+
+@navy:              #002D72; // Base
+@navy-80:           #33578E; // 80% tint
+@navy-50:           #7F96B8; // 50% tint
+@navy-20:           #CCD5E3; // 20% tint
+
+@pacific:           #0072CE; // Base
+@pacific-80:        #328ED8; // 80% tint
+@pacific-50:        #7FB8E6; // 50% tint
+@pacific-20:        #CCE3F5; // 20% tint
+
+@teal:              #005E5D; // Base
+@teal-80:           #337E7D; // 80% tint
+@teal-50:           #7FAEAE; // 50% tint
+@teal-20:           #CCDFDF; // 20% tint
+
+
+// Unofficial colors not part of the Official Palette,
+// but are in use for easier coding semantics or UX needs.
+
+// Active state of destructive action buttons.
+@dark-redorange:    #9C301B;
+
+@white:             #FFFFFF;
```

Comment:
Fixed in https://github.com/anselmbradford/generator-cf/commit/0732cf2c93b3f53733b6a8c4d0c507bc5409d9ac